### PR TITLE
Update version on docker pipeline

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: update_version.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Update Version on Docker Files
+
+env:
+  PRJ_VER: ${{ github.event.inputs.project_version }}
```
Comment (hanefi) [inline update_version.yml:4]: Can we have a longer name here?
```suggestion
  PROJECT_VERSION: ${{ github.event.inputs.project_version }}
```

File: update_version.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+
```
Comment (hanefi) [inline update_version.yml:7]: nit, remove extra spaces
```suggestion

```

File: update_version.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone Tools branch
+        run: git clone --branch tools-transition-update-docker https://github.com/citusdata/tools.git
```
Comment (hanefi) [inline update_version.yml:23]: I see that we need changes on different branches of tools repo. Let's make sure we change them into tags when the transition is complete

File: update_version.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        
+      - name: Install Python requirements
+        run: python -m pip install -r tools/python/requirements.txt
+        
+      - name: Update Docker Files
+        run: python -m python.update_docker --prj_ver ${PRJ_VER}
```
Comment (hanefi) [inline update_version.yml:29]: I do not think this path is correct. I think you need `tools` in the path as well

Comment (hanefi) [reply update_version.yml:29]: Also I am not comfortable with having directories called `python` and `bash` in our tools repository. The top directories used to be named after their purpose, not the language the files are written in. I think it makes more sense to move these two folders into `automated_packaging` folder there.

**Review by hanefi — COMMENTED** [review]
I think all Github Action Workflow definitions should be placed under `.github/workflows` directory. Please move the file there

File: update_version.yml (hunk 4 above)
Comment (gurkanindibay) [reply update_version.yml:29]: automated packaging folder is copied under /usr/bin/ which is unnecessary for our case.
But I totally agree with you to change the directory name

File: update_version.yml (hunk 3 above)
Comment (hanefi) [inline update_version.yml:23]:
```suggestion
      - name: Install Tools
        run: |
        	git clone --branch tools-transition-update-docker https://github.com/citusdata/tools.git
        	make -C tools install
```

You also need to install the scripts I think

File: update_version.yml (hunk 4 above)
Comment (hanefi) [reply update_version.yml:29]: You cloned the tools repository but did not run `make install`. See https://github.com/citusdata/docker/pull/218/files#r613963456

Also let me make a clarification:

> automated packaging folder is copied under /usr/bin/ which is unnecessary for our case.

All the files in the automated_packaging except Makefile is copied. If you create some folders, they will not be copied for example. You will need to make some changes in the Makefiles to actually install python and bash scripts on separate directories.

See `man install` and https://github.com/citusdata/tools/blob/develop/automated_packaging/Makefile for more info

Comment (gurkanindibay) [reply update_version.yml:29]: @hanefi I suggest going on different directory other than automated_packaging until I complete the transition process. Conceptually seperating directiories makes my life easier when developing solutions. I renamed python to  package_automation

File: update_version.yml (hunk 3 above)
Comment (gurkanindibay) [reply update_version.yml:23]: In this case there is no need to install scripts but I will add the package_automation directory into PYTHONPATH library

**Review by hanefi — APPROVED** [review]

**Pull request merged**